tests/test_isrFunctions.py

```diff
@@ -0,0 +1,416 @@
+        """Expect number of mask pixels with SAT marked to be non-zero.
+        """
+        corrMaskedImage = ipIsr.saturationCorrection(self.mi, 200, 2.0,
+                                                     growFootprints=2, interpolate=True,
+                                                     maskName='SAT')
+        numBit = countMaskedPixels(corrMaskedImage, "SAT")
+        self.assertEqual(numBit, 40800)
+
+    def test_saturationCorrectionNoInterpolate(self):
+        """Expect number of mask pixels with SAT marked to be non-zero.
+        """
+        corrMaskedImage = ipIsr.saturationCorrection(self.mi, 200, 2.0,
+                                                     growFootprints=2, interpolate=False,
+                                                     maskName='SAT')
+        numBit = countMaskedPixels(corrMaskedImage, "SAT")
+        self.assertEqual(numBit, 40800)
+
+    def test_trimToMatchCalibBBox(self):
+        """Expect bounding boxes to match.
+        """
+        darkExp = isrMock.DarkMock().run()
+        darkMi = darkExp.getMaskedImage()
+
+        nEdge = 2
+        darkMi = darkMi[nEdge:-nEdge, nEdge:-nEdge, afwImage.LOCAL]
+        newInput = ipIsr.trimToMatchCalibBBox(self.mi, darkMi)
+
+        self.assertEqual(newInput.getImage().getBBox(), darkMi.getImage().getBBox())
+
+    def test_darkCorrection(self):
+        """Expect round-trip application to be equal.
+        Expect RuntimeError if sizes are different.
+        """
+        darkExp = isrMock.DarkMock().run()
+        darkMi = darkExp.getMaskedImage()
+
+        mi = self.mi.clone()
+
+        # The `invert` parameter controls the direction of the
+        # application.  This will apply, and un-apply the dark.
+        ipIsr.darkCorrection(self.mi, darkMi, 1.0, 1.0, trimToFit=True)
+        ipIsr.darkCorrection(self.mi, darkMi, 1.0, 1.0, trimToFit=True, invert=True)
+
+        self.assertMaskedImagesAlmostEqual(self.mi, mi, atol=1e-3)
+
+        darkMi = darkMi[1:-1, 1:-1, afwImage.LOCAL]
+
+        with self.assertRaises(RuntimeError):
+            ipIsr.darkCorrection(self.mi, darkMi, 1.0, 1.0, trimToFit=False)
+
+    def test_biasCorrection(self):
+        """Expect smaller median image value after.
+        Expect RuntimeError is sizes are different.
+        """
+        biasExp = isrMock.BiasMock().run()
+        biasMi = biasExp.getMaskedImage()
+
+        mi = self.mi.clone()
+        ipIsr.biasCorrection(self.mi, biasMi, trimToFit=True)
+        self.assertLess(computeImageMedianAndStd(self.mi.getImage())[0],
+                        computeImageMedianAndStd(mi.getImage())[0])
+
+        biasMi = biasMi[1:-1, 1:-1, afwImage.LOCAL]
+
+        with self.assertRaises(RuntimeError):
+            ipIsr.biasCorrection(self.mi, biasMi, trimToFit=False)
+
+    def test_flatCorrection(self):
+        """Expect round-trip application to be equal.
+        Expect RuntimeError if sizes are different or if scaling type isn't known.
```

Comment:
How does this test test these two things? I only see the one `assertRaises` below.

Reply:
This message was leftover from before I split the tests in two.  Updated.